src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -29,62 +29,89 @@
-		<time
-			datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
-			class="tribe-common-a11y-visual-hide"
-		>
-			<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
-		</time>
-		<a href="<?php echo esc_url( $event->permalink ) ?>" class="tribe-events-calendar-month__multiday-event-inner">
-			<?php if ( $event->featured ) : ?>
-				<em
-					class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
-					aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-					title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-				></em>
-			<?php endif; ?>
-			<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
-				<?php echo get_the_title( $event ); ?>
-			</h3>
-		</a>
+	<article <?php tribe_classes( $event_classes ); ?> data-event-id="<?php echo esc_attr( $event->ID ); ?>">
+		<div class="tribe-events-calendar-month__multiday-event--hidden">
+			<time
+				datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
+				class="tribe-common-a11y-visual-hide"
+			>
+				<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
+			</time>
+			<a
+				href="<?php echo esc_url( $event->permalink ) ?>"
+				<?php if ( $display_tooltip ) : ?>
+					data-js="tribe-events-tooltip"
+					data-tooltip-content="#tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
+					aria-describedby="tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
+				<?php endif; ?>
+			>
+				<?php if ( $event->featured ) : ?>
+					<em
+						class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
+						aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+						title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+					></em>
+				<?php endif; ?>
+				<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
+					<?php echo get_the_title( $event ); ?>
+				</h3>
+			</a>
+		</div>
+	<?php if ( $should_display ) : ?>
+		<div <?php tribe_classes( $multiday_classes ); ?>>
+			<div class="tribe-events-calendar-month__multiday-event-inner">
+				<?php if ( $event->featured ) : ?>
+					<em
+						class="tribe-events-calendar-month__multiday-event-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
+						aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+						title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
+					></em>
+				<?php endif; ?>
+				<h3 class="tribe-events-calendar-month__multiday-event-title tribe-common-h8">
+					<?php echo get_the_title( $event ); ?>
+				</h3>
+			</div>
+		</div>
+		<?php $this->template( 'month/calendar-body/day/calendar-events/calendar-event/tooltip', [ 'event' => $event ] ); ?>
```

Comment:
the `$should_display` logic should be different from `$should_display_tooltip` logic.